proto/CHANGELOG.md

```diff
@@ -0,0 +1,119 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.32.0](https://github.com/eigerco/celestia-tendermint-rs/releases/tag/celestia-tendermint-proto-v0.32.0) - 2024-01-12
+
+This is the first release of the celestia-tendermint-rs, fork of tendermint-rs.
+
+### Added
+- *(proto)* allow serialization of Event ([#9](https://github.com/eigerco/celestia-tendermint-rs/pull/9))
+- use protobuffers from celestia-core ([#1](https://github.com/eigerco/celestia-tendermint-rs/pull/1))
+
+### Fixed
+- *(proto)* wrap values in options in bytes serializers ([#12](https://github.com/eigerco/celestia-tendermint-rs/pull/12))
+- *(celestia)* commit height serialization as number
+
+### Other
+- pre-release cleanups ([#16](https://github.com/eigerco/celestia-tendermint-rs/pull/16))
+- update prost to 0.12.0
+- remove unneded clones in proto encoding
+- Remove unneeded allocations in serializers ([#6](https://github.com/eigerco/celestia-tendermint-rs/pull/6))
+
+### Other - inherited
+- Prepare release for v0.32.0 ([#1314](https://github.com/eigerco/celestia-tendermint-rs/pull/1314))
```

Comment:
In this section the `https://github.com/eigerco/celestia-tendermint-rs` should be replaced with `https://github.com/informalsystems/tendermint-rs`.